src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -1,11 +1,131 @@
+  font-style: italic;
+`;
+const StyledActions = styled.div``;
+
+const StyledFormControl = styled(FormControl)`
+  width: 30%;
+  margin-top: 1rem;
+`;
+
+const StyledFileInfo = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
 
 interface FileCardProps {
-  file: any;
+  file: File;
+  removeFile: () => void;
+  updateFile: (newFile: File) => void;
 }
 
-const FileCard: React.FC<FileCardProps> = ({ file }) => {
-  return <p>{file.name}</p>;
+const FileCard: React.FC<FileCardProps> = ({ file, removeFile, updateFile }) => {
+  const { t } = useTranslation('publication');
+
+  return (
+    <Box>
+      <StyledTitle>{file.name}</StyledTitle>
+      <StyledDescription>
+        {t('files_and_license.uploaded_size', { size: Math.round(file.data.size / 1000) })}
+      </StyledDescription>
+
+      <FormControlLabel
+        control={
+          <Checkbox
+            color="primary"
+            checked={file.administrativeContract}
+            onChange={() =>
+              updateFile({
+                ...file,
+                administrativeContract: !file.administrativeContract,
+              })
+            }
+          />
+        }
+        label={t('files_and_license.administrative_contract')}
+      />
+
+      {!file.administrativeContract && (
+        <StyledFileInfo>
+          <StyledFormControl>
+            <FormLabel component="legend">{t('files_and_license.select_version')}</FormLabel>
+            <RadioGroup
+              aria-label="version"
+              value={file.isPublished}
+              onChange={event =>
+                updateFile({
+                  ...file,
+                  isPublished: event.target.value === 'published',
+                })
+              }>
+              <FormControlLabel
+                value="accepted"
+                control={<Radio color="primary" checked={file.isPublished !== null && !file.isPublished} />}
+                label={t('files_and_license.accepted_version')}
+              />
+              <FormControlLabel
+                value="published"
+                control={<Radio color="primary" checked={!!file.isPublished} />}
+                label={t('files_and_license.published_version')}
+              />
+            </RadioGroup>
+          </StyledFormControl>
+
+          <StyledFormControl>
+            <MuiPickersUtilsProvider utils={DateFnsUtils}>
+              <KeyboardDatePicker
+                inputVariant="outlined"
+                label={t('files_and_license.embargo_date')}
+                onChange={value =>
+                  updateFile({
+                    ...file,
+                    embargoDate: value,
+                  })
+                }
+                value={file.embargoDate}
+                autoOk
+              />
+            </MuiPickersUtilsProvider>
+          </StyledFormControl>
+
+          <StyledFormControl>
+            <TextField select variant="outlined" label={t('files_and_license.license')}></TextField>
```

Comment:
skal det være noe children her?

Reply:
ja, skal være en liste med lisenser, som kommer som en egen sak senere. La det bare til for å vise hvordan det blir.